plugin/trino-password-authenticators/src/main/java/io/trino/plugin/password/ldap/LdapAuthenticator.java

```diff
@@ -65,7 +65,8 @@ public LdapAuthenticator(LdapAuthenticatorClient client, LdapAuthenticatorConfig
         this.client = requireNonNull(client, "client is null");
 
         this.userBindSearchPatterns = ldapAuthenticatorConfig.getUserBindSearchPatterns();
-        this.groupAuthorizationSearchPattern = Optional.ofNullable(ldapAuthenticatorConfig.getGroupAuthorizationSearchPattern());
+        this.groupAuthorizationSearchPattern =
+                Optional.ofNullable(ldapAuthenticatorConfig.getGroupAuthorizationSearchPattern());
```

Comment:
can we revert this change ?

Reply:
we could .. but this AuthorizationSearchPattern is not used on LDAP Group Provider.
it was a copy and paste from the LDAP Auth Provider.
so i would suggest no need to revert as it was not used ?

Reply:
Sorry I meant moving `Optional.ofNullable(ldapAuthenticatorConfig.getGroupAuthorizationSearchPattern());` to a new line

Reply:
Or we could have a dedicated commit for the formatting changes.